#!/usr/bin/env python
PACKAGE = "range_image_viewer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angular_resolution", double_t, 0, "", 1, 1, 30)
gen.add("max_angle_width", double_t, 0, "", 360, 30, 360)
gen.add("max_angle_height", double_t, 0, "", 180, 30, 180)
gen.add("sensor_pos_x", double_t, 0, "", 0, -100, 100)
gen.add("sensor_pos_y", double_t, 0, "", 0, -100, 100)
gen.add("sensor_pos_z", double_t, 0, "", 0, 0, 100)

exit(gen.generate(PACKAGE, "range_image_viewer", "RangeImg"))